### PR TITLE
[Range slider] Remove ndl fallbacks

### PR DESCRIPTION
**Pull request by aveline**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/polaris-ux/issues/550

### WHAT is this pull request doing?
Removes fallback references to `newDesignLanguage` in the Range slider component

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
- [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by aveline** [comment]
For troubleshooting the tests in Playground

Playground

```
   const mockProps: DualThumbProps = {
    id: 'RangeSlider',
    value: [0, 50],
    min: 0,
    max: 50,
    step: 1,
    output: true,
    disabled: false,
    onChange: noop,
    label: 'Dual thumb range slider',
  };

  return (
    <Page title="Playground">
      {/* <div style={{width: '124px'}}> */}
      <DualThumb {...mockProps} />
      {/* </div> */}
    </Page>
  );
```

**Review by alex-page — APPROVED** [review]

**Comment by alex-page** [comment]
Awesome work @aveline well done fixing up the tests ✨

File: src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx [hunk 1]
```diff
@@ -81,10 +79,7 @@ export function SingleThumb(props: SingleThumbProps) {
     disabled && styles.disabled,
   );
 
-  const inputClassNames = classNames(
-    styles.Input,
-    newDesignLanguage && styles.newDesignLanguage,
-  );
+  const inputClassNames = classNames(styles.Input);
```
Comment (kyledurand) [inline src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx:82]: We don't need the classNames function anymore

**Review by kyledurand — APPROVED** [review]
Nice work! Just one super minor comment around removing the classNames function.

**Pull request merged**